src/com/lftechnology/java/training/dipak/employeemanagement/dao/EmployeeDao.java

```diff
@@ -0,0 +1,251 @@
+
+	/**
+	 * <p>This method is used to add a row in database. The data is obtained
+	 * from the Employee object. This method returns the number of rows affected
+	 * in the database</p>
+	 * 
+	 * @param e
+	 * @return count
+	 */
+	@Override
+	public int addEmployee(Employee e) {
+
+		Connection con = DbConnection.setConnection();
+		int count = 0;
+		if (con == null) {
+			LOGGER.info("Connection Setup Failed");
+			return count;
+		}
+		PreparedStatement pst = null;
+		String sql =
+			"insert into employee (username,password,isTerminated,fullname,department,address,role) values(?,?,?,?,?,?,?)";
+		try {
+			pst = con.prepareStatement(sql);
+
+			pst.setString(1, e.getUserName());
+			pst.setString(2, e.getPassword());
+			pst.setBoolean(3, e.getIsTerminated());
+			pst.setString(4, e.getFullName());
+			pst.setString(5, e.getDepartment());
+			pst.setString(6, e.getAddress());
+			pst.setString(7, e.getRole().toString());
+
+			System.out.println(pst);
+
+			count = pst.executeUpdate();
+
+		}
+		catch (MySQLIntegrityConstraintViolationException e1) {
+			LOGGER.severe(
+				"User name already exits. Please enter another username.");
+		}
+		catch (SQLException sq) {
+			LOGGER.log(Level.SEVERE, "Exception::{0}", sq);
+		}
+
+		return count;
+	}
+
+	/**
+	 * <p>This method is used to view records present in database. The filters
+	 * are used for filtering the records. It return the result set.</p>
+	 * 
+	 * @param e
+	 * @return rs
+	 */
+	@Override
+	public ResultSet viewEmployee(Employee e) {
+
+		ResultSet rs = null;
+		StringBuilder sql = null;
+		Connection con = DbConnection.setConnection();
+		if (con == null) {
+			LOGGER.info("Connection Setup Failed");
+			return rs;
+		}
+		PreparedStatement pst = null;
+
+		sql = new StringBuilder("select * from employee where 1=1 ");
+		if (!e.getFullName().equals("")) {
+			sql.append("and fullname='");
+			sql.append(e.getFullName());
+			sql.append("'");
+		}
+		if (!e.getAddress().equals("")) {
+			sql.append(" and address='");
+			sql.append(e.getAddress());
+			sql.append("'");
+		}
+		if (!e.getDepartment().equals("")) {
+			sql.append(" and department='");
+			sql.append(e.getDepartment());
+			sql.append("'");
+		}
+
+		try {
+			pst = con.prepareStatement(sql.toString());
+			rs = pst.executeQuery();
+
+		}
+		catch (SQLException e1) {
+			LOGGER.log(Level.SEVERE, "Exception::{0}", e1);
+		}
+
+		// sql="select * from employee where fullname=? and department=? and
+		// address=?";
+		//
+		// try {
+		// pst=con.prepareStatement(sql);
+		// if(!e.getFullName().equals("")) {
+		// pst.setString(1, e.getFullName());
+		// }else {
+		// pst.setString(1, "?");
+		// }
+		// if(!e.getAddress().equals("")) {
+		// pst.setString(3, e.getAddress());
+		// }else {
+		// pst.setString(3, "?");
+		// }
+		// if(!e.getDepartment().equals("")) {
+		// pst.setString(2, e.getDepartment());
+		// }else {
+		// pst.setString(2, "?");
+		// }
+		//
+		// System.out.println(pst);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)